tests/modules/servers/resources/test_endpoints.py

```diff
@@ -0,0 +1,59 @@
+"""
+Test server endpoints
+"""
+import pytest
+
+@pytest.mark.parametrize('http_method,http_path', (
+    ('GET', '/servers/'),
+    ('POST', '/servers/'),
+    ('GET', '/servers/1'),
+    ('PUT', '/servers/1'),
+    ('DELETE', '/servers/1'),
+    ('POST', '/servers/1/ping'),
+    ('POST', '/servers/1/batch_request'),
+))
+
+def test_general_return(http_method, http_path, flask_app_client):
```

Comment:
All the tests "assert response is not None" is way too broad.

Nicer would be checking how many servers there are, then adding one and seeing if there is an extra server for example.

Reply:
We are currently finishing fixing the tests, the main hold-up was in overwriting the authentication via Auth0, so the tests can be done locally in isolation. Will be fixed in the upcoming sprint request.